index.html

```diff
@@ -2,8 +2,13 @@
   <head>
     <title>TITLE GOES HERE</title>
   </head>
+
   <body>
     <div id="app"></div>
     <script src="src/entry-point/index.tsx"></script>
+    <link
```

Comment:
nit: put stylesheets under <head>!

Reply:
hey - please clarify by "under" 😄

Reply:
omg… bad github formatting. I meant to write:

> nit: put stylesheets under `<head>`!

Reply:
Just noticed something. I think between this `<link>` and the following in `styles.tsx`

```
  @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
  @import url('https://fonts.googleapis.com/css?family=Libre+Baskerville&display=swap');
  @import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
```

we only need to use one of them!

Reply:
I thought so too, which is why the previous PR was just shipped with the css imports - but I noticed it wasn't actually using the right fonts. After adding it here, the proper fonts are loading 😄